Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -8417,6 +8418,20 @@
 		FEE43FD21E67AFC60077D6D1 /* WebInspectorInterruptDispatcher.messages.in */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text; path = WebInspectorInterruptDispatcher.messages.in; sourceTree = "<group>"; };
 /* End PBXFileReference section */
 
+/* Begin PBXFileSystemSynchronizedBuildFileExceptionSet section */
```

Comment:
I'm not sure this is necessary?

Reply:
This is what Xcode did when I added `WebExtensionFrameIdentifierCocoa.mm`. 🤷‍♂️